code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -857,3 +857,17 @@
 			return
 	clear_products(holder, step_volume_added)
 	holder.my_atom.audible_message("<span class='notice'>[icon2html(holder.my_atom, viewers(DEFAULT_MESSAGE_RANGE, src))] The reaction gives out a fizz, teleporting items everywhere!</span>")
+
+/datum/chemical_reaction/ants
+	results = list(/datum/reagent/ants = 2)
+	required_reagents = list(/datum/reagent/ants = 1, /datum/reagent/consumable/sugar = 3)
+
+/datum/chemical_reaction/big_ant
+	required_reagents = list(/datum/reagent/ants = 20, /datum/reagent/medicine/omnizine = 20, /datum/reagent/medicine/strange_reagent = 5)
```

Comment:
Why omnizine and strange reagent? Is there something else that's more flavorful you could put here maybe? like ahhh apple juice or something?